app/forms/edit_product_form.rb

```diff
@@ -0,0 +1,34 @@
+class EditProductForm < TranslatableBaseForm
```

Comment:
Using a form object felt more right that putting some of these methods directly onto Product. It did mean I had to do some extra work to get the form itself to get translations properly.